Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/UnitData.xml

```diff
@@ -23576,4 +23604,128 @@
         <HotkeyAlias value=""/>
         <ScoreKill value="200"/>
     </CUnit>
+    <CUnit id="AP_PrimalIgniter">
+        <DeathRevealRadius value="3"/>
+        <Race value="Zerg"/>
+        <Mob value="Multiplayer"/>
+        <FlagArray index="PreventDestroy" value="1"/>
+        <FlagArray index="UseLineOfSight" value="1"/>
+        <FlagArray index="ArmySelect" value="1"/>
+        <PlaneArray index="Ground" value="1"/>
+        <Collide index="Ground" value="1"/>
+        <Collide index="ForceField" value="1"/>
+        <Collide index="Locust" value="1"/>
+        <Collide index="Small" value="1"/>
+        <Attributes index="Armored" value="1"/>
+        <Attributes index="Biological" value="1"/>
+        <LifeStart value="250"/>
+        <LifeMax value="250"/>
+        <LifeArmor value="2"/>
+        <LifeRegenRate value="0.2734"/>
+        <LifeArmorName value="Unit/LifeArmorName/ZergGroundArmor"/>
+        <ShieldArmorName value="Unit/ShieldArmorName/ZergWildMutation"/>
+        <Speed value="2.9492"/>
+        <Acceleration value="1000"/>
+        <LateralAcceleration value="46.0625"/>
+        <StationaryTurningRate value="999.8437"/>
+        <TurningRate value="999.8437"/>
+        <Sight value="9"/>
+        <Food value="-3"/>
+        <CostCategory value="Army"/>
+        <CostResource index="Minerals" value="150"/>
+        <CostResource index="Vespene" value="50"/>
+        <AttackTargetPriority value="20"/>
+        <AbilArray Link="stop"/>
+        <AbilArray Link="attack"/>
+        <AbilArray Link="move"/>
+        <AbilArray Link="AP_PrimalIgniterBurrowDown"/>
+        <AbilArray Link="AP_PrimalIgniterMergeable"/>
+        <AbilArray Link="ProgressRally"/>
+        <BehaviorArray Link="AP_MalignantCreepFix"/>
+        <WeaponArray Link="AP_PrimalIgniterCone"/>
+        <CardLayouts>
+            <LayoutButtons Face="Move" Type="AbilCmd" AbilCmd="move,Move" Row="0" Column="0"/>
+            <LayoutButtons Face="Stop" Type="AbilCmd" AbilCmd="stop,Stop" Row="0" Column="1"/>
+            <LayoutButtons Face="MoveHoldPosition" Type="AbilCmd" AbilCmd="move,HoldPos" Row="0" Column="2"/>
+            <LayoutButtons Face="Attack" Type="AbilCmd" AbilCmd="attack,Execute" Row="0" Column="4"/>
+            <LayoutButtons Face="MovePatrol" Type="AbilCmd" AbilCmd="move,Patrol" Row="0" Column="3"/>
+            <LayoutButtons Face="BurrowDown" Type="AbilCmd" AbilCmd="AP_PrimalIgniterBurrowDown,0" Row="2" Column="4"/>
+            <LayoutButtons Face="AP_RapidRegeneration" Type="Passive" Row="1" Column="0"/>
+            <LayoutButtons Face="AP_PrimalIgniterFireBreath" Type="Passive" Requirements="AP_HavePrimalIgniterFireBreath" Row="1" Column="1"/>
+            <LayoutButtons Face="AP_GlialReconstitution" Type="Passive" Requirements="AP_HaveHotSRoachSpeed" Row="1" Column="2"/>

```

Comment:
Glial doesn't increase Ingiter's movement speed:
![Snímek obrazovky pořízený 2024-07-05 11-06-08](https://github.com/Ziktofel/Archipelago-SC2-data/assets/3447808/4ab4d5d0-5318-42f9-a15b-93b417f2dc59)
![Snímek obrazovky pořízený 2024-07-05 11-06-03](https://github.com/Ziktofel/Archipelago-SC2-data/assets/3447808/905d6ea3-4373-42bc-86bd-84fb970d9095)

Reply:
Turns out there are two different copies of Glial Reconstitution in the upgrade code... I'll see if there are any references to the 2nd one (which is where I put the igniter speed) and if not, I'll just delete it